core/xml.ts

```diff
@@ -190,7 +190,7 @@ export function blockToDom(
 
   const commentText = block.getCommentText();
   if (commentText) {
-    const comment = block.getIcon(COMMENT_TYPE) as CommentIcon;
+    const comment = block.getIcon(IconType.COMMENT) as CommentIcon;
```

Comment:
I think you should be able to remove the casts now right? (here and below)

Reply:
Done!